package.json

```diff
@@ -93,6 +94,7 @@
     "grunt-jsonlint": "1.0.7",
     "grunt-newer": "1.2.0",
     "grunt-nsp": "2.2.0",
+    "grunt-remarkable": "1.0.0",
```

Comment:
Seems like we should remove **marked** and **grunt-marked** dependencies as well.